### PR TITLE
For #9342 feat(project): Add configuration folder with yaml files for each mobile application

### PR DESCRIPTION
**Pull request by eliserichards**
Because

- We want to specify the following for each of the mobile apps:
   - repo link
   - path to the fml file for the repo
   - a mapping for the version tags (so that we can find the appropriate FML for a specific version)

This commit

- Adds a new folder, `/experimenter/features/configs` that contains a file for each mobile application
   - This locations--`/features`--also contains the feature definitions that map to the `NimbusFeature` API
- Adds files for: 
   - ios
   - focus ios
   - fenix
   - focus android

----
We don't need anything for Klar, do we?

### CODE REVIEW

**Review by jaredlockhart — COMMENTED** [review]
Should these be in separate yamls?  Or folded into the [application config constants](https://github.com/mozilla/experimenter/blob/main/experimenter/experimenter/experiments/constants.py#L60-L175)?  Where will these be consumed?

**Comment by jaredlockhart** [comment]
Also I should've pointed out, right now these paths are [defined](https://github.com/mozilla/experimenter/blob/main/Makefile#L56-L60) and [consumed](https://github.com/mozilla/experimenter/blob/main/Makefile#L91-L95) directly in the Makefile, but as we add (many!) more applications, we'll want a more streamlined way to add and configure applications, which I believe is what this PR is driving towards, and to that effect one option is to move it all into Python space and then move the FML integration commands also into Python space as a management command filed here https://github.com/mozilla/experimenter/issues/9286, though I pre emptively filed that and it may not be the correct approach and I defer to to @brennie to decide on an effective strategy.

File: experimenter/experimenter/features/configs/fenix.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+---
+fenix:
+  repo: @mozilla-mobile/firefox-android/tree/main/fenix
+  fmlPath: app/nimbus.fml.yaml
```
Comment (brennie) [inline experimenter/experimenter/features/configs/fenix.yaml:7]:
```suggestion
  repo: @mozilla-mobile/firefox-android
  fmlPath: fenix/app/nimbus.fml.yaml
```

File: experimenter/experimenter/features/configs/focus-android.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+---
+focus-android:
+  repo: @mozilla-mobile/firefox-android/tree/main/focus-android
+  fmlPath: app/nimbus.fml.yaml
```
Comment (brennie) [inline experimenter/experimenter/features/configs/focus-android.yaml:7]:
```suggestion
  repo: @mozilla-mobile/firefox-android
  fmlPath: focus-android/app/nimbus.fml.yaml
```

**Review by brennie — CHANGES_REQUESTED** [review]

**Review by brennie — APPROVED** [review]

**Pull request merged**